lang/py/avro/schema.py

```diff
@@ -214,6 +236,19 @@ def validate(self, datum):
         """
         raise Exception("Must be implemented by subclasses.")
 
+    @abc.abstractmethod
+    def to_canonical_json(self, names):
+        """
+        Converts the schema object into its Canonical Form
+        http://avro.apache.org/docs/current/spec.html#Parsing+Canonical+Form+for+Schemas
+        """
+        raise NotImplementedError("to_canonical_json must be implemented by subclasses")
```

Comment:
Doesn't the abc take care of this?

Reply:
It does 👍 
Addressed in eacfdd6a0cd6de9b4e15701d08d9e994f09be310